src/target/tag.cc

```diff
@@ -66,12 +79,252 @@ Target TargetTag::AddTag(String name, Map<String, ObjectRef> config, bool overri
   return Target(config);
 }
 
-/**********  Register Target tags  **********/
-
-TVM_REGISTER_TARGET_TAG("nvidia/rtx2080ti")
-    .set_config({
-        {"kind", String("cuda")},
-        {"arch", String("sm_75")},
-    });
+/**********  Register Target tags  ***********/
 
+  TVM_REGISTER_CUDA_TAG("nvidia/tesla-k80", "sm_37", 49152, 65536);
```

Comment:
Wait, shall we remove the indent here too?

Reply:
Fixed.